### ISSUE TITLE
No updates ? 

### ISSUE THREAD

**Issue by RiadReyBouazza** [iHelp101/XInsta#131]
Hello kind sir, I noticed that this module isn't officially getting any update, I managed to hunt some test apk links from issues in the past but I'm not finding anything now. 
Can you please tell me how to get updates ? Thanks a lot.

**Comment by RiadReyBouazza** [comment]
EDIT : the in-app update didn't show up for me in a long time, and I always doubted that it provided the actual latest. Hence why this " issue " is opened. I'd love to get testing apks. Thank you.

**Comment by iHelp101** [comment]
The latest APK is on GitHub (XInsta.apk). Regarding the update check do you have any issues on your network with GitHub? XInsta checks the Version.txt file to see if a new version is out. It is a very simple process done.

**Comment by RiadReyBouazza** [comment]
> The latest APK is on GitHub (XInsta.apk). Regarding the update check do you have any issues on your network with GitHub? XInsta checks the Version.txt file to see if a new version is out. It is a very simple process done.

Thanks for the reply. All I was saying is that everytime I try the apk it doesn't work, so I thought maybe the update check isn't giving me the latest apk and that I have to do it manually. Is this a valid concern or no ?

**Comment by iHelp101** [comment]
Yes, does it downloads the APK? And then the installation fails? Or it never shows the new APK? Is there any chance you can provide a screenshot recording on this process? I want to verify for sure where it is failing

**Comment by RiadReyBouazza** [comment]
No it's not that zerio

**Comment by RiadReyBouazza** [comment]
No it's not that serious ! If that is what you thought then sorry for wasting your time. 
The story is that I was stuck on version 2.1.0 which never showed me there was an update . Till one day I decided to come here to the github issues and look for answers. That's where I found you linked some person a test apk and I downloaded to use it and it worked ! So from that day I had this doubt that : do I trust the automatic updates ? Or do I always have to come look for the actual latest version ? 
Again sorry for wasting your time man.

**Comment by iHelp101** [comment]
No problem, I wanted to see if the auto-updater was having issues. The automatic updater should work. I did take a while to release the last update, so it was only available via the test URL (Google Drive). I am working on updating the module more often to fix bugs. Currently, I am working on a new automatic hook finder.

**Comment by RiadReyBouazza** [comment]
> No problem, I wanted to see if the auto-updater was having issues. The automatic updater should work. I did take a while to release the last update, so it was only available via the test URL (Google Drive). I am working on updating the module more often to fix bugs. Currently, I am working on a new automatic hook finder.

That Would be amazing ! I think nothing works for me rn with latest stable instagram release. So that would be a game changer. Thanks a lot and keep up the great work !

**Comment by ayush7gl** [comment]
@iHelp101 what's the latest instagram version supported?? How to check it btw?
